mne/coreg.py

```diff
@@ -1245,3 +1248,656 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
+            rot=self._default_parameters[:3],
+            tra=self._default_parameters[3:6],
+            sca=self._default_parameters[6:9],
+        )
+
+    def _setup_bem(self):
+        # find high-res head model (if possible)
+        high_res_path = _find_head_bem(self._subject, self._subjects_dir,
+                                       high_res=True)
+        low_res_path = _find_head_bem(self._subject, self._subjects_dir,
+                                      high_res=False)
+        if high_res_path is None and low_res_path is None:
+            raise RuntimeError("No standard head model was "
+                               f"found for subject {self._subject}")
+        if high_res_path is not None:
+            self._bem_high_res = _read_surface(high_res_path)
+            logger.info(f'Using high resolution head model in {high_res_path}')
+        else:
+            self._bem_high_res = _read_surface(low_res_path)
+            logger.info(f'Using low resolution head model in {low_res_path}')
+        if low_res_path is None:
+            # This should be very rare!
+            warn('No low-resolution head found, decimating high resolution '
+                 'mesh (%d vertices): %s' % (len(self._bem_high_res.surf.rr),
+                                             high_res_path,))
+            # Create one from the high res one, which we know we have
+            rr, tris = decimate_surface(self._bem_high_res.surf.rr,
+                                        self._bem_high_res.surf.tris,
+                                        n_triangles=5120)
+            # directly set the attributes of bem_low_res
+            self._bem_low_res = complete_surface_info(
+                dict(rr=rr, tris=tris), copy=False, verbose=False)
+        else:
+            self._bem_low_res = _read_surface(low_res_path)
+
+    def _setup_fiducials(self, fids):
+        _validate_type(fids, (str, dict, list))
+        # find fiducials file
+        if fids == 'auto':
+            fid_files = _find_fiducials_files(self._subject,
+                                              self._subjects_dir)
+            if len(fid_files) > 0:
+                # Read fiducials from disk
+                fid_filename = fid_files[0].format(
+                    subjects_dir=self._subjects_dir, subject=self._subject)
+                logger.info(f'Using fiducials from: {fid_filename}.')
+                fids, _ = read_fiducials(fid_filename)
+            else:
+                fids = 'estimated'
+
+        if fids == 'estimated':
+            logger.info('Estimating fiducials from fsaverage.')
+            fids = get_mni_fiducials(self._subject, self._subjects_dir)
+
+        if isinstance(fids, list):
+            fid_coords = _fiducial_coords(fids)
+        else:
+            assert isinstance(fids, dict)
+            fid_coords = np.array([fids['lpa'], fids['nasion'], fids['rpa']],
+                                  dtype=float) / 1e3  # convert to meters
```

Comment:
Conversion to meters should not be necessary, all units in MNE data structures should be SI (here, meters). This seems like a bug, especially since:
```
>>> import mne
>>> mne.io.read_fiducials('mne/data/fsaverage/fsaverage-fiducials.fif')[0][-1]['r'])
array([ 0.08436285, -0.02850276, -0.04127743], dtype=float32)
```
I'll change this to assume and be documented as `m`